src/test/java/teammates/test/pageobjects/AppPage.java

```diff
@@ -698,7 +698,7 @@ public AppPage verifyHtml(String filePath) {
             filePath = TestProperties.TEST_PAGES_FOLDER + filePath;
         }
         String actual = getPageSource();
-        
+        actual = processPageSourceForGodMode(actual);
```

Comment:
yup mainly for that case. But also moving forward we are planning to change all the comparisons from regex to normal string comparison. This can only be done if the actual html is processed